File: music-album-webapp/Entities/Track.cs
namespace music_album_webapp.Entities;

public class Track
{
    public int Id { get; set; }
    public string Title { get; set; }
    public string Author { get; set; }
    public int ReleaseYear { get; set; }
    public int Duration { get; set; } // seconds
    public int AlbumId { get; set; }
    public virtual Album Album { get; set; }
}

File: music-album-webapp/ClientApp/src/app/models/Track.ts
export class Track {

  constructor(id: number, title: string, author: string, releaseYear: string | null, duration: string | null) {
    this.id = id;
    this.title = title;
    this.author = author;
    this.releaseYear = releaseYear;
    this.duration = duration;
  }

  id: number;
  title: string;
  author: string;
  releaseYear: string | null;
  duration: string | null;
}
